qemu/tests/qemu_guest_agent.py

```diff
@@ -366,6 +365,18 @@ def gagent_setsebool_value(self, value, params, vm):
             self.test.error("Set boolean virt_qemu_ga_read_nonsecurity_files "
                             "failed.")
 
+    @error_context.context_aware
+    def log_persistence(self, params, session, vm):
+        """
+        Create new log directory and make it as log persistence.
+        """
+        error_context.context("Make logs persistent.", logging.info)
+        session.cmd(self.params["cmd_create_log_dict"])
+        session.cmd(self.params["cmd_chgrp_chmod"])
+
+        logging.info("Restart systemd-journald to take effect.")
+        session.cmd(self.params["cmd_restart_journald"])
```

Comment:
You might send the commands together within one `session.cmd()` by join them together with `&&`. Although each way works.

Reply:
warp commands together in .cfg file?

Reply:
@6-dehan Yes.

Reply:
okay,got it .

Reply:
@vivianQizhu